apps/calendar/src/components/events/timeEvent.tsx

```diff
@@ -95,8 +95,23 @@ function getStyles(uiModel: EventUIModel, isDraggingTarget: boolean, hasNextStar
   };
 }
 
+function isDraggableEvent({
+  uiModel,
+  isReadOnlyCalendar,
+  isDraggingTarget,
+  hasNextStartTime,
+}: {
+  uiModel: EventUIModel;
+  isReadOnlyCalendar: boolean;
+  isDraggingTarget: boolean;
+  hasNextStartTime: boolean;
+}) {
+  const { model } = uiModel;
+  return !(isReadOnlyCalendar || model.isReadOnly || isDraggingTarget || hasNextStartTime);
```

Comment:
```suggestion
  return !isReadOnlyCalendar && !model.isReadOnly && !isDraggingTarget && !hasNextStartTime;
```
반전해서 이렇게는 어떠신가요? 단순 취향차이같긴 합니다..

리턴문 한줄 안띄워도 에러가 없나보네요. 🤔

Reply:
처음 완성된걸 그렇게 했다가 인텔리제이 단축키로 뒤집었어요. 사람이 읽는다는 측면에서 후자가 나으니 원복시킬게요.